tools/check_env.py

```diff
@@ -54,8 +55,81 @@ def check_backend():
     logger.info(f'openvino_is_avaliable: {openvino_apis.is_available()}')
 
 
+def get_version(s):
+    return ''.join([c for c in s if c.isdigit() or c == '.'])
+
+
+def get_op(s):
+    return ''.join([c for c in s if c in '><='])
+
+
+def check_codebase():
+    ops = {
+        '>=': (lambda x, y: x >= y),
+        '==': (lambda x, y: x == y),
+        '<=': (lambda x, y: x <= y),
+        '': (lambda x, y: True)
+    }
+    with open('../requirements/optional.txt', 'r') as file:
+        try:
+            import mmcls
+        except ImportError:
+            mmcls_version = None
+        else:
+            mmcls_version = mmcls.__version__
+        requirement = file.readline()
```

Comment:
Not safe to check packages one by one.
The package order in `optional.txt` might change in the future.

Reply:
Version requirements may change according to [PEP ](https://www.python.org/dev/peps/pep-0440/#version-specifiers) and this check may fail at some cases.